lib/streamlit/__init__.py

```diff
@@ -111,6 +110,7 @@
 import json as _json
 import numpy as _np
 
+from streamlit.util import functools_wraps
```

Comment:
Inside `__init__.py` always prepend internal functions with `_`, so users know they're not part of our API.